.pre-commit-config.yaml

```
@@ -1,56 +1,70 @@
 repos:
-#####
-# Basic file cleanliness
--   repo: https://github.com/pre-commit/pre-commit-hooks
+  #####
+  # Basic file cleanliness
+  - repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v4.5.0
     hooks:
-    -   id: check-added-large-files
-    -   id: check-yaml
-    -   id: check-toml
-    -   id: end-of-file-fixer
-    -   id: mixed-line-ending
-    -   id: trailing-whitespace
-#####
-# Python
--   repo: local
-    hooks:
-    -   id: generate-rst-placeholders
+      - id: check-added-large-files
+      - id: check-yaml
+      - id: check-toml
+      - id: end-of-file-fixer
+      - id: mixed-line-ending
+      - id: trailing-whitespace
+  #####
+  # Python
+  - repo: local
+    hooks:
+      - id: generate-rst-placeholders
         name: Tutorials .rst placeholders (local run only)
-        entry: ./.pre-commit-rst-placeholder.sh
+        entry: ./hook_scripts/pre-commit-rst-placeholder.sh
         language: script
         files: 'model/docs/.*\.qmd'
--   repo: https://github.com/psf/black
+  - repo: https://github.com/psf/black
     rev: 23.10.0
     hooks:
-    -   id: black
-        args: ['--line-length', '79']
--   repo: https://github.com/PyCQA/isort
+      - id: black
+        args: ["--line-length", "79"]
+  - repo: https://github.com/PyCQA/isort
     rev: 5.12.0
     hooks:
-    -   id: isort
-        args: ['--profile', 'black',
-               '--line-length', '79']
--   repo: https://github.com/astral-sh/ruff-pre-commit
+      - id: isort
+        args: ["--profile", "black", "--line-length", "79"]
+  - repo: https://github.com/astral-sh/ruff-pre-commit
     rev: v0.1.0
     hooks:
-    -   id: ruff
--   repo: https://github.com/numpy/numpydoc
+      - id: ruff
+  - repo: https://github.com/numpy/numpydoc
     rev: v1.7.0
     hooks:
-    -   id: numpydoc-validation
-#####
-# Secrets
--   repo: https://github.com/Yelp/detect-secrets
+      - id: numpydoc-validation
+  #####
+  # Quarto
+  - repo: local
+    hooks:
+      - id: format-qmd-python
+        name: Format Python in QMD
+        entry: python hook_scripts/quarto_python_formatter.py "-q --line-length 79"
```

This works for me and I think is more readable 
```suggestion
        entry: python hook_scripts/quarto_python_formatter.py 
        args: ["-q --line-length 79"]
```

---

I was following advice from pre-commit docs https://pre-commit.com/#arguments-pattern-in-hooks

> When creating local hooks, there's no reason to put command arguments into [args](https://pre-commit.com/#config-args) as there is nothing which can override them -- instead put your arguments directly in the hook [entry](https://pre-commit.com/#hooks-entry).